tests/providers/test_braket_backend.py

```diff
@@ -141,6 +141,23 @@ def test_local_backend_circuit_shots0(self):
         self.assertEqual(statevector[2], 0.0 + 0.0j)
         self.assertEqual(statevector[3], 1.0 + 0.0j)
 
+    def test_meas_level_2(self):
+        """Check that there's no error for asking for classified measurement results."""
+        backend = BraketLocalBackend(name="default")
+        circuit = QuantumCircuit(1, 1)
+        circuit.h(0)
+        circuit.measure(0, 0)
+        backend.run(circuit, shots=10, meas_level=2)
+
+    def test_meas_level_1(self):
+        """Check that there's an exception for asking for raw measurement results."""
+        backend = BraketLocalBackend(name="default")
+        circuit = QuantumCircuit(1, 1)
+        circuit.h(0)
+        circuit.measure(0, 0)
+        with self.assertRaises(exception.QiskitBraketException):
+            backend.run(circuit, shots=10, meas_level=0)
```

Comment:
nit: should it be meas_level=1, according to the function name?

Reply:
ty!